src/style.css

```diff
@@ -0,0 +1,62 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+.form {
+  position: relative;
+  margin: 20px 0;
+}
+
+.img__loupe {
+  color: #687c94;
+  width: 19px;
+  height: 19px;
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+.img__loupe__small {
+  color: #687c94;
+  width: 11px;
+  height: 11px;
+  position: absolute;
```

Comment:
Don't repeat yourself. Make another class (img__loupe), add it to both small and normal image and style it.